src/main/java/com/github/romankh3/image/comparison/ImageComparison.java

```diff
@@ -86,6 +86,17 @@ public void setThreshold(int threshold) {
      * @return the result of the drawing.
      */
     public BufferedImage compareImages() throws IOException {
+        return compareImages(1);
```

Comment:
Please, provide default value as a variable to the class. Add for it getter and setter.